app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+            # Include default devise modules.
+            devise :database_authenticatable, :registerable
+                    #:recoverable, :rememberable, :trackable, :validatable,
```

Comment:
I recommend adding trackable as well on real use cases. You can keep it this way if you like.